CMakeLists.txt

```diff
@@ -238,7 +239,7 @@ function(build_GenomicsDB_executable_common target)
     if(HTSLIB_SOURCE_DIR)
         add_dependencies(${target} htslib)
     endif()
-    target_link_libraries(${target} genomicsdb ${TILEDB_LIBRARY} ${HTSLIB_LIBRARY} ${CMAKE_DL_LIBS})
+    target_link_libraries(${target} genomicsdb ${TILEDB_LIBRARY} ${JAVA_JVM_LIBRARY} ${HTSLIB_LIBRARY} ${CMAKE_DL_LIBS})
```

Comment:
Should it be
if(USE_HDFS)
   target_link_libraries(JAVA_JVM..)
endif()

Reply:
Yes, we could change it to that. Do you want me to take a pass before you merge?

Reply:
Done
One question - is the CMAKE_CL_LIBS needed?

Reply:
It was needed before because of dynamic loading, but with the changed way of including hdfs, it is still needed indirectly. I am moving that to under USE_HDFS too.

Reply:
DONE. Note that CMAKE_CL_LIBS seems to be needed by libcrypto, so leaving it in the general target_link_libraries stmt.